code/parse/sexp.cpp

```diff
@@ -24452,11 +24451,6 @@ int eval_sexp(int cur_node, int referenced_node)
 				sexp_supernova_stop(node);
 				break;
 
-			case OP_SET_MOTION_DEBRIS:
-				sexp_val = SEXP_TRUE;
-				sexp_set_motion_debris(node);
```

Comment:
It looks like you left `sexp_set_motion_debris` in the code even though you deleted its invocation.